src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -94,8 +94,16 @@ InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
         return new (compiler, CMK_Inlining) ProfilePolicy(compiler, isPrejitRoot);
     }
 
-    // Use the default policy by default
-    return new (compiler, CMK_Inlining) DefaultPolicy(compiler, isPrejitRoot);
+    const bool useExtendedDefaultPolicy = JitConfig.JitExtendedDefaultPolicyModel() != 0;
+
+    if (!useExtendedDefaultPolicy || (!isPrejitRoot && compiler->opts.IsReadyToRun()))
+    {
+        // DefaultPolicy is better for AOT in terms of code-size.
+        return new (compiler, CMK_Inlining) DefaultPolicy(compiler, isPrejitRoot);
```

Comment:
CoreRT may well want the CQ benefits from the more aggressive policy, so this may need adjusting.